payloads/payloads_common.go

```diff
@@ -0,0 +1,384 @@
+package payloads
```

Comment:
Can be in the future but a name for this package is potentially `runners`, matches quite a the functions below (which we could rename to simplify) e.g. `runners.Dir()`.